docs/reference/ml/df-analytics/apis/get-trained-models.asciidoc

```diff
@@ -67,7 +67,9 @@ Specifies whether the included model definition should be returned as a JSON map
 
 `for_export`::
 (Optional, boolean)
-include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=for-export]
+Indicates if certain fields should be removed from the model configuration on
```

Comment:
Same comment as https://github.com/elastic/elasticsearch/pull/63093#discussion_r508388796

Reply:
Can the `ml-shared.asciidoc` snippit be written in a general enough way to be used here too?

Shouldn't `for_export` be deprecated for trained models and switch to the `exclude_generated` flag? Or it that for another PR

Reply:
@davidkyle I think keeping it `for_export` makes sense for now as its use case is really only for export (no clone makes sense for trained models)

Reply:
I'd much prefer the same flag was used for the same purpose in all the APIs. Trained models will look like an oddity when transforms, AD jobs and DFA all use `exclude_generated` 

The use case shouldn't define the parameter name, in future there may be another reason to get the model without the generated fields e.g. copy and update the model with further training. I like `exclude_generated` as it describes the action that takes place rather than one possible use case.